day_7/fizzbuzz.rb

```diff
@@ -0,0 +1,103 @@
+# original version
```

Comment:
Nice job implementing Fizzbuzz a few different ways Shane! There are often multiple ways to solve the same problem in programming, and it's our job to find the best one! Which one of these do you think is the most flexible? I encourage you to attempt the other day-7 challenges if you have time this weekend. You can add a time constraint of 1-3 hours for an extra challenge, and practice for what you will face weekly at Turing. Overall great work Shane!